frontend/js/player_adapter_HTML5.js

```diff
@@ -77,98 +80,130 @@ define(["jquery",
 
                 targetElement.style.width = "100%";
                 targetElement.style.height = "100%";
-
+                targetElement.preload = "auto";
+                //targetElement.muted = true;
+                
+                window.Hls = Hls;
                 mediaElementPlayer = new mejs.MediaElementPlayer(targetElement, {
+                    renderers: ['html5', 'native_hls'],
+                    hls: {
+                        //debug: true
+                    },
                     alwaysShowControls: true,
                     stretching: "fill",
-                    success: function (mediaElement) {
+                    success: function (wrapper) {
+                        mediaElement = wrapper;
+
+                        if (Hls !== undefined) {
```

Comment:
This entire block is unnecessary, no?

Reply:
That block is a logger to present that the HLS media was correctly attached, is purely optional and can be use for info log.

I can delete it if you want.

Reply:
I'm generally not a friend of excessive logging just for the sake of it. I would take it out for now. Should we ever feel the need to log more stuff, we should probably build a proper logging system where we can control what is logged at runtime.

Reply:
I think the Idea to a more complete log solution should be the best answer, for the moment I deleted that fragment.